src/api/patron-codes.md

```diff
@@ -125,4 +125,10 @@ Request][polling] using the transacting APIs.
 }
 ```
 
+{% h4 Error Responses %}
+
+| Status |          Code              |                   Description                |
+| :----- | :--------------------------| :--------------------------------------------|
+| 403    | {% break _ INVALID_LUHN %} | Luhn checksum digit doesn't pass, try again. |
```

Comment:
can we say "CHECKSUM_FAILED" and remove "try again"?

Reply:
done :)